tests/core/test_utils.py

```diff
@@ -52,3 +52,28 @@ def test_select_maximum_cpu_core(self, cpu_count_mock):
                 )
 
             self.assertEqual(utils.select_maximum_cpu_core("invalid"), 1)
+
+    def test_is_weak_hash_algo(self):
+        for algo in ("md2", "md4", "md5", "sha1"):
+            self.assertEqual(utils.is_weak_hash_algo(algo), True)
```

Comment:
Here and and everywhere else
```suggestion
            self.assertTrue(utils.is_weak_hash_algo(algo))
```